test/DebuggerCommon/async.js.dbg.baseline

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "callStack": [
+      {
+        "line": 10,
+        "column": 4,
+        "sourceText": "return await af1(10)",
+        "function": "af2"
+      },
+      {

```

Comment:
Ok, it seems PDM enables library stack frames and our default value is false so ChakraCore doesn't give it. For now its fine but we may need to expose a functionality to enable this in ChakraCore in future.